src/main/scala/util/Timer.scala

```diff
@@ -94,14 +94,14 @@ class GenericTimerCfgRegIFC (
     RegFieldGroup(s"${prefix}cfg", Some(s"${prefix} Configuration"),
       Seq(
         RegField(scaleWidth, RegReadFn(read.scale), RegWriteFn((v, d) => writeFn(v, d, write.scale, write_scale)), descs.scale),
-        RegField(8-scaleWidth),
+        RegField(8-scaleWidth,RegFieldDesc.reserved),
```

Comment:
I'm pretty sure that this happens automatically.

Reply:
See https://github.com/chipsalliance/rocket-chip/blob/45f3c98e392f8224bd40e189f837f92e6d745b6e/src/main/scala/regmapper/RegField.scala#L113